app/views/posts/_expanded.html.erb

```diff
@@ -144,15 +144,8 @@
                 <% end %>
               </div>
             <% end %>
-            <% if (post_type.has_license && post.license.present?) || been_edited %>
+            <% if been_edited %>
```

Comment:
Because I removed the license from this view and moved it to the user card.

Reply:
That seems potentially misleading - it could give the impression of being a license for the user or for all that user's content, when it's in fact the license for the post... I'd either leave it here, or we should be very careful about how it's displayed, and share screenshots to agree on.

Reply:
![screenshot of the user card](https://user-images.githubusercontent.com/21335202/102686903-2f342a80-41eb-11eb-96cb-33baa692d063.png)

Shouldn't be misleading with this display (relates more to created than to the user), but I'll also make the tooltip more explicit.